### ISSUE TITLE
Integrate Copyright agent

### ISSUE THREAD

**Issue by jeffmcaffer** [clearlydefined/crawler#151]
Once https://github.com/fossology/fossology/pull/1155 is merged we can add support for the Copyright agent as we did with Nomos

**Comment by maxhbr** [comment]
The related PR in fossology is now merged into upstream.

**Comment by dabutvin** [comment]
this has been integrated
